client/src/components/Register/Introduction.js

```diff
@@ -1,10 +1,11 @@
+/* eslint-disable import/no-cycle */
```

Comment:
I'm seeing this rule being disabled inline a lot, seems like we should just switch it off in npm package instead?